pkg/apis/migration/v1alpha1/migmigration_types.go

```diff
@@ -34,6 +34,7 @@ type MigMigrationSpec struct {
 	QuiescePods     bool                  `json:"quiescePods,omitempty"`
 	KeepAnnotations bool                  `json:"keepAnnotations,omitempty"`
 	Verify          bool                  `json:"verify,omitempty"`
+	Cancel          bool                  `json:"cancel,omitempty"`
```

Comment:
The spec is declarative so this should be the adjective `canceled` not the verb `cancel`.
Please use the _American English_ spelling.

Reply:
What happens if a user sets Canceled to false after it's been set to true?

Reply:
@sseago Similar behaviour as for MigPlan `closed` state. If nothing mentions that it was cancelled. No conditions/spec fields - it will just continue from the `Phase` it is in. Typically after cancellation being done - it is `Completed` phase. For that reason there is a condition check as well, which will prevent accidental run, when a `cancel` being set at least once.